dragonchain/broadcast_processor/broadcast_processor.py

```diff
@@ -232,6 +251,7 @@ def get_all_notification_endpoints(level: str) -> set:
             await broadcast_functions.save_unfinished_claim(block_id)
             continue
         claim_chains = chain_id_set_from_matchmaking_claim(claim, current_level)
+        broadcast_receipt_wait_time_l5 = get_l5_wait_time(claim["properties"]["metadata"]["properties"]["dcId"])  # see: matchmaking schema
```

Comment:
definitely shouldn't get this value unless we already know this is an L5

Reply:
Good point; added conditional.